.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateExtractor.cs

```diff
@@ -306,9 +306,16 @@ private List<Token> DurationWithBeforeAndAfter(string text, DateObject reference
             foreach (var er in durationEr)
             {
                 // if it is a multiple duration and its type is not equal to Date than skip it.
-                if (er.Data != null && er.Data.ToString() != Constants.MultipleDuration_Date)
+                if (er.Data != null && er.Data.ToString().StartsWith("multipleDuration") && er.Data.ToString() != Constants.MultipleDuration_Date)
```

Comment:
Cases like "more than 3 days from today" will result in er.Data equals to "more".
The original condition can't handle this case, so add more accurate restrictions here.